[DEVBUG] Pyinstaller macOS build sometimes fails

### Type of bug

Code - general bugs related to how the game runs and responds to you.

### Description

idk if this is happening for anyone else, but when I synced my fork I got a notification that the macOS Pyinstaller build failed, but this commit had all checks passed on upstream.
The main annoyance from this is getting the failed notification, but I'd love if we could get build to stop failing.
links here to the failed builds:
https://github.com/rustykitty/clangen/actions/runs/12634603763/job/35202703784 on https://github.com/rustykitty/clangen/commit/f8f80c11b27420ebbd704c4af8412d9871691028, https://github.com/rustykitty/clangen/actions/runs/12672866043/job/35317866388 and on https://github.com/rustykitty/clangen/commit/fdab408d3153acd461bfd5437aab7a8e1110b06b.

### Grade

Unimportant - This bug has little to no impact on play.

### Steps to reproduce the behavior

idk, I just synced my fork

### Patrol ID + Outcome Print (if applicable)

N/A

### Commit # or Game Version Number

see above

### Additional context

idk if you can see the logs, so I'll just paste them here.
```
Run rm -r dist/Clangen
  rm -r dist/Clangen
  npm install -g appdmg
  appdmg appdmg.json Clangen_macOS64.dmg
  shell: /bin/bash -l {0}
  env:
    gpg_private_key: 
    gpg_private_key_passphrase: 
    update_api_push_token: 
    pythonLocation: /Users/runner/hostedtoolcache/Python/3.11.9/x64
    PKG_CONFIG_PATH: /Users/runner/hostedtoolcache/Python/3.11.9/x64/lib/pkgconfig
    Python_ROOT_DIR: /Users/runner/hostedtoolcache/Python/3.11.9/x64
    Python2_ROOT_DIR: /Users/runner/hostedtoolcache/Python/3.11.9/x64
    Python3_ROOT_DIR: /Users/runner/hostedtoolcache/Python/3.11.9/x64
    VERSION_NUMBER: fdab408d3153acd461bfd5437aab7a8e1110b06b
    RELEASE_CHANNEL: 

added 50 packages in 11s

1 package is looking for funding
  run `npm fund` for details
[ 1/21] Looking for target...                [ OK ]
[ 2/21] Reading JSON Specification...        [ OK ]
[ 3/21] Parsing JSON Specification...        [ OK ]
[ 4/21] Validating JSON Specification...     [ OK ]
[ 5/21] Looking for files...                 [ OK ]
[ 6/21] Calculating size of image...         [ OK ]
[ 7/21] Creating temporary image...          [ OK ]
[ 8/21] Mounting temporary image...          [ OK ]
[ 9/21] Making hidden background folder...   [ OK ]
[10/21] Copying background...                [ OK ]
[11/21] Reading background dimensions...     [ OK ]
[12/21] Copying icon...                      [SKIP]
[13/21] Setting icon...                      [SKIP]
[14/21] Creating links...                    [ OK ]
[15/21] Copying files...                     [ OK ]
[16/21] Making all the visuals...            [ OK ]
[17/21] Blessing image...                    [ OK ]
[18/21] Unmounting temporary image...        [FAIL]
[19/21] Removing temporary image...          [ OK ]
[20/21] Removing target image...             [ OK ]

Error: Command failed: hdiutil detach /Volumes/Clangen
hdiutil: detach failed - No such file or directory


Error: Process completed with exit code 1.
```

### Screenshots

_No response_

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -165,7 +165,7 @@ jobs:
         if: startsWith(matrix.os, 'macos')
         run: |
           rm -r dist/Clangen
-          npm install -g archanyhm/appdmg
+          npm install -g archanyhm/node-appdmg
           appdmg appdmg.json ${{ matrix.artifact_name }}
       - uses: actions/upload-artifact@v4
         with:

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -165,7 +165,7 @@ jobs:
         if: startsWith(matrix.os, 'macos')
         run: |
           rm -r dist/Clangen
-          npm install -g appdmg
+          npm install -g archanyhm/appdmg
           appdmg appdmg.json ${{ matrix.artifact_name }}
       - uses: actions/upload-artifact@v4
         with:

